verify/verify-pylint.sh

```diff
@@ -17,6 +17,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+export PYLINTHOME=$(mktemp -d)
+export PATH="${TEST_SRCDIR}/pylint:${PATH}"
```

Comment:
I'm not particularly interested in making it work outside of bazel. I don't think it's worth supporting two build and test systems. That said, it's easy enough to do, so if you insist...